app/views/proposals/details/_approvals.html.haml

```diff
@@ -7,7 +7,7 @@
             Approvals
         .content-content.column
           - index = 0
-          - proposal.steps.each do |step|
+          - proposal.steps.order('position ASC').each do |step|
```

Comment:
I believe this is already available as

```
proposal.individual_steps
```

which should be in position order.